planning/src/parsing/sexpr.rs

```diff
@@ -354,6 +401,26 @@ mod tests {
         assert_eq!(&formatted, output);
     }
 
+    #[test]
+    fn parsing_string() {
+        formats_as("(a \"b c\" d)", "(a \"b c\" d)");
+        formats_as("(a \"(b c)\" d)", "(a \"(b c)\" d)");
+        formats_as("(a b); \"(a b)\"", "(a b)");
+    }
```

Comment:
Missing test cases for `;` and `\n` in a string.

Reply:
Resolved in 5587ae9